ledger-tool/src/main.rs

```diff
@@ -1856,6 +1856,11 @@ fn main() {
             .about("Prints the ledger's shred hash")
             .arg(&hard_forks_arg)
             .arg(&max_genesis_archive_unpacked_size_arg)
+            .arg(&accounts_index_bins)
+            .arg(&accounts_index_limit)
+            .arg(&disable_disk_index)
+            .arg(&accountsdb_verify_refcounts)
+            .arg(&accounts_db_skip_initial_hash_calc_arg)
```

Comment:
I don't love all this duplication. We could possibly throw these into an array and then for each command that needs the args, do `.args(accounts_db_args)`. However, that doesn't stop someone from adding a new arg to an individual command instead of this common array. Any thoughts on this aspect welcome.

Reply:
Sure, the impl you provided is pretty similar to what I had in mind. But, any thoughts about this specific piece:

> However, that doesn't stop someone from adding a new arg to an individual command instead of this common array.

I don't like the duplication as I initially mentioned, but I dislike leaving the door open for inconsistency when adding new args. Duplication aside, at least everything is consistent at the moment. Thoughts on shipping this PR as is and maybe circling back to add something like we've discussed in this thread if we find a solution we like?

Reply:
yeah. we can ship this pr. And leave the refactoring to a follow up pr.

I don't have a good solution to enforce consistency when adding new args too. I will think about it and will let you know if I come up with some ideas.